app/src/main/res/values-v21/styles.xml

```diff
@@ -8,4 +9,12 @@
         <item name="android:windowSharedElementExitTransition">@android:transition/move</item>
         <item name="android:windowActionModeOverlay">true</item>
     </style>
+
+    <!-- Techically valid for v17 -->
+    <style name="ActionButtonStyle" parent="Widget.AppCompat.ActionButton">
+        <item name="android:minWidth">0dip</item>
+        <item name="android:paddingStart">8dip</item>
+        <item name="android:paddingEnd">8dip</item>
+    </style>
```

Comment:
Reduce icon spacing

Reply:
you can move it to just styles.xml. It will probably tell you about Start/End not being available but you can try with silence it with tools ignore.